josm-presets/de-signals-eso.xml

```diff
@@ -195,6 +197,14 @@ See http://wiki.openstreetmap.org/wiki/OpenRailwayMap for details.
 						delete_if_empty="true" />
 					<key key="railway:signal:main:states"
```

Comment:
Here you have still a <key> for the states, for the other signals you removed that.